src/obfl-specification.html

```diff
@@ -548,6 +548,11 @@ <h4 id="L608" class="include">template</h4>
           <dd>An expression.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd>
+      <p>In the use-when condition, the the commonly available variables as
+        listed with the evaluate element can be used.</p>
```

Comment:
The link is present. It is generated automatically. Please have a look at this temporary available [preview](https://rambags.home.xs4all.nl/obfl/obfl-specification.html#L608).

> I don't like that these variables are defined in the "evaluate" section.

I think the best approach is to remove the overview of all variables from the `evaluate` element and put it in a general section.

Typo is fixed.

Reply:
> The link is present.

Cool. I didn't realize that.

I really question the usefulness of such a general section, for the reasons I described in the comment above. What you win from this is so limited because there are pretty much no variables that are available for every `use-when` attribute (basically only $volume and $volumes, which are very easy to explain), and when you also consider which variables actually make sense to use in each situation you end up with no general available variables at all.

Just give each `expression`, `use-when` and `cost` attribute its own "Variables" section, and for `expression` have a overview list that links to relevant sections for more info (like is already the case).